volumes/miovision/sql/create-table-classifications.sql

```diff
@@ -3,7 +3,12 @@
 CREATE TABLE miovision.classifications (
 	classification_uid serial,
 	classification text,
-	location_only boolean -- for peds and bikes, where movement isn't available, only which leg they were observed on
+	location_only boolean, -- for peds and bikes, where movement isn't available, only which leg they were observed on
+	class_type_id smallint,
```

Comment:
The table in `miovision_new` and `miovision_api` both include a `class_type TEXT` column. In `miovision_new` there is also a separate table `class_types` which I'm assuming only contains the unique texts.